packages/common/src/store/premium-content/slice.ts

```diff
@@ -82,6 +84,12 @@ const slice = createSlice({
     resetLockedContentId: (state) => {
       state.lockedContentId = null
     },
+    setPurchaseContentId: (state, action: PayloadAction<IdPayload>) => {
```

Comment:
Kind of what I thought at first, and I think I was hesitant because of how that value gets manipulated from multiple places. But I may need to merge the two modals together anyway. Alright if I leave it for now?